coremltools/models/neural_network/quantization_utils.py

```diff
@@ -420,7 +420,7 @@ def _quantize_nn_spec(spec, nbits, qm, **kwargs):
     ]
 
     # Bump up to appropriate spec version if required
-    if nbits > 8 and nbits != 16:
+    if nbits and nbits > 8 and nbits != 16:
```

Comment:
What does `nbits and nbits` accomplish?

Reply:
If nbits is None, the if conditional is False in python 2 (expected behavior) but it errors out in python 3 (as None cannot be compared to int). 
By making this change we only continue with the condition check if nbits is not None.

Reply:
I think this could be accomplished by `if nbits is not None and nbits > 8 and nbits != 16:`. I think that would also be less cryptic.

Reply:
Agreed.